errai-bom/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.jboss.integration-platform</groupId>
     <artifactId>jboss-integration-platform-parent</artifactId>
-    <version>8.0.0.CR3</version>
+    <version>8.1.0-SNAPSHOT</version>
```

Comment:
@mbiarnes @tiagobento it is my first wf upgrade.

How we make the build green without setting a snapshot version here?

Reply:
I don't know. Ideally we shouldn't use SNAPSHOT versions.. There's a `8.1.0.CR1` artifact on `JBoss Releases`, are we able to use that?

Reply:
Nevermind, just saw https://github.com/errai/errai/pull/356.